starlight/starlight/starlight.vcxproj

```diff
@@ -27,7 +27,6 @@
     <ClInclude Include="core\math\vec3.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="core\math\main.cpp" />
```

Comment:
I think the last call we made was to have both of our mains be included in master, and on respective branches we remove the other person's main from the solution. Include this main again before pushing up this branch (change should propogate to other vcxproj files changed in this PR too)

Reply:
Weird, I re-included the math main after I was finished testing my MemMgr code. I'll add that line manually to the vcxproj file.